src/plugins/plot/MctPlot.vue

```diff
@@ -999,12 +1004,22 @@ export default {
                 this.removeStatusListener();
             }
 
+            this.plotContainerResizeObserver.disconnect();
+
             this.openmct.time.off('clock', this.updateRealTime);
             this.openmct.time.off('bounds', this.updateDisplayBounds);
             this.openmct.objectViews.off('clearData', this.clearData);
         },
         updateStatus(status) {
             this.$emit('statusUpdated', status);
+        },
+        handleWindowResize() {
+            if (this.offsetWidth !== this.$parent.$refs.plotWrapper.offsetWidth) {
+                this.offsetWidth = this.$parent.$refs.plotWrapper.offsetWidth;
+                this.config.series.models.forEach((series) => {
```

Comment:
Hey just a small thing, you can shorten this (similarly to how you do it elsewhere in the code) for consistency, to (this.loadSeriesData, this).

Reply:
Done.